lib/master.js

```diff
@@ -302,14 +310,14 @@ function restart() {
   }
 
   function startOne() {
-    self.setSize(self.options.size + 1);
+    self.setSize(self.options.size + 1, true);
 
     self.once('resize', stopOne);
   }
 
   function stopOne() {
     waitForStability(function() {
-      self.setSize(self.options.size - 1);
+      self.setSize(self.options.size);
```

Comment:
Reading this out of context it looks like a bug.. is `self.size !== self.options.size` a guaranteed precondition of this function? Otherwise, the `'resize'` event below would logically never happen.

Maybe the context is more clear when not viewing it as a diff? I think I'd like a comment explaining why this isn't a bug, if there's a short enough explanation.